src/vt_flash/newton.jl

```diff
@@ -194,7 +261,8 @@ function vt_flash_newton!(
     nmol::AbstractVector,
     volume::Real,
     RT::Real;
-    gtol::Real,
+    chemrtol::Real,
+    pressrtol::Real,
```

Comment:
If `|mu'_i - mu''_i| / RT < tol` and `|P' - P''| V / sum(nmol) RT < tol` are used, the tolerances could be the same.